internal/builder/builder.go

```diff
@@ -108,16 +108,16 @@ type options struct {
 }
 
 // FromImage constructs a builder from a builder image
-func FromImage(img imgutil.Image) (*Builder, error) {
-	return constructBuilder(img, "", true)
+func FromImage(img imgutil.Image, runImage string, registry string) (*Builder, error) {
```

Comment:
Rather than adding arguments, it might be cleaner to add a `BuilderOption` to support this, e.g.,

```
type BuilderOption func(*options) error

type options struct {
	flatten bool
	exclude []string
	runImage string     <- new
}

func WithRunImage(name string) BuilderOption {
	return func(o *options) error {
		o.runImage = name
		return nil
	}
}
```